internal/controllers/app_controller.go

```diff
@@ -63,10 +63,10 @@ type Helm interface {
 	DeleteChart(appName string) error
 }
 
-// +kubebuilder:rbac:groups=theketch.io,resources=apps,verbs=get;list;watch;create;update;patch;delete
-// +kubebuilder:rbac:groups=theketch.io,resources=apps/status,verbs=get;update;patch
-// +kubebuilder:rbac:groups=theketch.io,resources=frameworks,verbs=get;list;watch;create;update;patch;delete
-// +kubebuilder:rbac:groups=theketch.io,resources=frameworks/status,verbs=get;update;patch
+// +kubebuilder:rbac:groups=theketch.io;shipa.io,resources=apps,verbs=get;list;watch;create;update;patch;delete
```

Comment:
I think it is OK to keep only theketch.io here because it used to generate config/rbac/* and we are good to convert the ketch's config to shipa's config manually or with a script.